src/docs/product/accounts/sso/azure-sso.mdx

```diff
@@ -57,7 +57,17 @@ description: "Learn about the Azure Active Directory Single Sign-On (SSO)."
     - First Name: `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname`
 
     - Last Name: `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname`
+    
+    - For more details about mappings for custom configs, see [The Role of Claims](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/technical-reference/the-role-of-claims).
```

Comment:
This doesn't make structural sense as a bullet of step since it's not one of AzureAD keys. I've tried to format it so that it remains indented--so that it's clear the information relates to the larger step--but just without a bullet point. 

```suggestion
      For more details about mappings for custom configs, see [The Role of Claims](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/technical-reference/the-role-of-claims).
```